crates/oxc_parser/src/js/function.rs

```diff
@@ -216,7 +216,13 @@ impl<'a> Parser<'a> {
         let ident = self.ast.binding_identifier(param);
         let pattern = self.ast.binding_pattern(ident, None, false);
         let params_span = self.end_span(params_span);
-        let formal_parameter = self.ast.formal_parameter(params_span, pattern, None, false, None);
+        let formal_parameter = self.ast.formal_parameter(
+            params_span,
+            pattern,
+            None,
+            false,
+            Vec::new_in(self.ast.allocator),
```

Comment:
Not sure if this is correct, not familiar with `oxc_allocator::Vec` 👍🏼

Reply:
AstBuilder exported some utils at the top, see `AstBuilder::new_vec`.